src/test.python.bridge.cpp

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+#include <memory>
+
+#include <Python.h>
+
+#include <openassetio/log/LoggerInterface.hpp>
```

Comment:
Could start a whole flame war about quotes vs angle brackets.

But, the point I'd make, is that it depends on _where_ you install libraries as to which is appropriate, especially on compilers that distinguish include search path command line arguments between user and system paths.

In Conan land, these files will _never_ be in a system location.
VCPKG they won't be either (although in their own /usr like structure).
rez, I don't know.

All I'm saying is that "it depends" whether this is considered a system library or not.

Reply:
My take on angle brackets vs. quotes is to reserve quotes for relative paths (since IIRC angle brackets don't work at all for that, at least on GCC).  There's a ticket (https://github.com/OpenAssetIO/OpenAssetIO/issues/424) over on the OpenAssetIO repo related to this.

Although Conan does not install into a system location, the include directories are typically added to the `gcc` command line using `-isystem`, so are at least treated in a similar way to system headers.

Reply:
Happy there's a ticket on it.
Don't envy the task of making everyone happy on a decision though. Sorry.